Makefile

```
@@ -96,11 +96,17 @@ install-rog_gui: target/$(TARGET)/$(BIN_ROG)
 
 install-program: install-asusd install-asus-shutdown install-asusctl install-asusd_user install-rog_gui
 
-install-data-rog_gui:
+install-data-rog_gui: target/$(TARGET)/$(BIN_ROG)
```

checkmake (0.3.2)

[warning] Target body for "install-data-rog_gui" exceeds allowed length of 5 lines (10).

(maxbodylength)